### ISSUE TITLE
[Screen Reader - Makecode - Feedback Survey] Narrator is reading wrong information at progress bar.

### ISSUE THREAD

**Issue by v-swasha** [Microsoft/pxt#2551]
**User Experience:**
User depend on Screen Reader will misguided if narrator in item mode is giving wrong information at progress bar.

**Test Environment:**
OS: RS2 Version 1703(OS Build 15063.483)
Platform: Edge.

**Repro Steps**:-
1. Navigate to url:- https://makecode.com/
2. Navigate to the Microbit section element and select Code control given on it.
3. Navigate to the Settings control lying in the header section on the page and select it.
4. Navigate to the Give feedback menu option listed under the Settings menu control and select it.
5. Navigate to the various controls lying in the pane opened and select Next.
6. Navigate to 'How likely..' page ,provide some input and select Next/Previous button control.
7. Navigate to 'Provide changes..' page ,provide some input and select Next/Previous button control.
8. Navigate to Progress bar in item mode(Caps + Left\Right) arrow Keys and verify the issue.

**Actual Result**:-
Narrator is reading as progress bar at 0, minimum Value 0 and Maximum Value 0.

**Expected Result**:-
Narrator should read as progress bar at 43, minimum Value 0 and Maximum Value 100.

**Note:**
This issue doesn't exists with NVDA

**MAS Reference** -
https://microsoft.sharepoint.com/teams/msenable/Documents/LCA/MAS%20-%20July%201,%202017%20Release/MAS%204.2.1%20%E2%80%93%20Object%20Information.docx?d=wa12f3779bdf140c2bbcc0e314ea94858

**Refer Screen Shot**
[MAS 4.2.1_MakeCode_Narrator is reading wrong info at Progress bar.zip](https://github.com/Microsoft/pxt/files/1179954/MAS.4.2.1_MakeCode_Narrator.is.reading.wrong.info.at.Progress.bar.zip)

**Comment by veler** [comment]
The give feedback website is an external website. We don't have the control on it. We cannot fix it.

**Comment by veler** [comment]
Needs to be forwarded to [Survey Monkey](https://www.surveymonkey.com).

**Comment by abchatra** [comment]
External bug with Survey Monkey
